lib/hammer_cli_foreman/hosts/common_update_options.rb

```diff
@@ -78,6 +85,7 @@ def request_params
         if action == :update
           params['host']['compute_attributes']['volumes_attributes'] = nested_attributes(option_volume_list) unless option_volume_list.empty?
           params['host']['interfaces_attributes'] = interfaces_attributes unless option_interface_list.empty?
+          HammerCLIForeman::Hosts::CommonUpdateOptions::set_image_uuid(params, options["option_image_id"])  if options["option_image_id"]
```

Comment:
Not sure if this is the best place but we should load cr_id somewhere. How about?
```ruby
options["option_image_id"]
  compute_resource_id = params['host']['compute_resource_id'] || load_compute_resource(options["option_id"]) || <rise some error>
  params['host']['compute_attributes']['image_id'] = image_uuid(compute_resource_id, options["option_image_id"])
end
```